### DISCUSSION TITLE
Prevent taskbar buttons from overlapping the clock on secondary displays

### DISCUSSION THREAD

**Discussion by stamminator** [ramensoftware/windhawk-mods#1100] [Mod ideas]
On secondary displays, buttons/labels on the taskbars display right over the clock on the right. It looks like a right margin is not being set properly.

![0UqHyPd93T](https://github.com/user-attachments/assets/32f88d66-9f3a-4198-a079-9b89d20671ed)

In my case, this is on a secondary display (the built-in laptop monitor) with 250% scaling, while my main monitor (via Thunderbolt dock) has 100% display scaling.

A Windhawk mod could make sure this overlap doesn't occur. Would this be better suited as a feature of the Windows 11 Taskbar Styler mod, or its own mod? Since this one looks simpler than the others I've pitched, I might take a crack at this one myself.

**Comment by m417z** [comment]
> Would this be better suited as a feature of the Windows 11 Taskbar Styler mod, or its own mod?

You can start with the taskbar styler, it doesn't require C++ programming knowledge and it's easier to experiment with. Not everything can be done via styles, though.
